app/javascript/packs/user.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function(){
+  $.ajaxSetup({
+    headers: {
+      'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content')
+    }
+  });
+
+  $('.delete-item-cart').click(function(){
+    let parent = $(this).parents('li');
+    let count_items_cart = parseInt($('.count-items-cart').text());
```

Comment:
js convention đặt tên theo kiểu camelCase nha 
```suggestion
    let countItemsCart = parseInt($('.count-items-cart').text());
```

Reply:
.